hhast-lint.json

```diff
@@ -24,23 +24,6 @@
       "extraLinters": [
         "Facebook\\HHAST\\DataProviderTypesLinter"
       ]
-    },
-    {
-      "patterns": ["src/Migrations/*"],
-      "linterConfigs": {
-        "Facebook\\HHAST\\HHClientLinter": {
-          "ignore": [5624, 5639]
-        }
-      }
-    }
-  ],
-  "linterConfigs": {
-    "Facebook\\HHAST\\HHClientLinter": {
-      "Comments[5607] (not an actual setting)":
-        " - 5607 is ignored because of false positives when comparing a generic to a typed value",
-      "Comments[5624&5639] (not an actual setting)":
-        " - 5624 and 5639 are ignored because they insist on using co(tra)variant generics. Could this break external consumers?",
-      "ignore": [5607, 5624, 5639]
     }
-  }
+  ]
```

Comment:
<3

Reply:
... for the whole file changes, not specifically the bracket :)

Reply:
Seems like I did something wrong when resolving conflicts